htseq2multx.pl

```diff
@@ -182,18 +186,22 @@ sub processOptions
     my $supported_version                 = [split(/\./,$FQMXVERSION)];
     if($problem =~ /./)
       {
+        help();
         help2();
-        print('ERROR: There is a problem with the fastq-multx executable: ',
-              "[$problem].  Use the --fastq-multx option described above to ",
-              "supply fastq-multx if it is missing from your PATH.\n");
+        print STDERR ('ERROR: There is a problem with the fastq-multx ',
+                      "executable: [$problem].  Use the --fastq-multx option ",
+                      'described above to supply fastq-multx if it is ',
+                      "missing from your PATH.\n");
```

Comment:
@jcmatese - I'd like your opinion on making this error come out on `STDERR`. I'm not sure if there was a reason to originally have it on `STDOUT`. I tried finding documentation that might explain it, but having not found one, I assumed this was simply an oversight and changed it to STDERR. Do you know of an instance where this could break anything downstream?

Reply:
This seems like a good change and shouldn't affect things unless there was already a problem. Might be good to run a test or two in staging after this gets pushed out just to make sure the errors are captured and passed along properly. Unless John knows of a reason this would cause and issue, I'd say merge and test later.

Reply:
Not that I know of.  I think both STDOUT and STDERR _might_ end up in the log, but I don't think it would "break" anything.

Reply:
I'm not terribly familiar with the testing in staging process, however testing that some errors come out on STDERR is in the test script in this package, which should be run during installation. Would that suffice?

Reply:
No, I think it would be prudent to run this through using HTSEQ to see that the error is propagated properly. I don't think that should hold anything up, but good to test the system that will execute and consume the output (now on STDERR) before this change gets too old.

Reply:
#10